packages/material-ui/src/RadioGroup/useRadioGroup.d.ts

```diff
@@ -0,0 +1,11 @@
+import { Context } from 'react';
+import { RadioGroupProps } from './RadioGroup';
+
+// shut off automatic exporting
+export {};
+
+type ContextFromPropsKey = 'name' | 'onChange' | 'value';
```

Comment:
Not sure from `ContextFromProps` means, but I copied it from the original `useFormControl`. I guess I could've inlined these keys to avoid having to name things.

Reply:
```jsx

function Component({ contextPropertyFromProp, basicProp }) {
	return <SomeContext.Provider value={{ contextPropertyFromProp }} />
}
```

It's basically what props passed to `FormControl` we expose via context.

> I guess I could've inlined these keys to avoid having to name things.

I should've done this, good point :+1:  Where those come from is irrelevant for the context and types.

Reply:
I'll inline here then.